File: PRAK110-2610817210017-Faris Ramadhan.py
import math
tinggi = 12
alas = 5
miring = math.sqrt(alas**2 + tinggi**2)

print(f"Diketahui :")
print(f"Alas = {alas}cm")
print(f"Tinggi = {tinggi}cm")
print(f"")
keliling = tinggi + alas + miring
luas = 0.5 * alas * tinggi
print(f"Jawab :")
print(f"Sisi A = {tinggi}cm")
print(f"Sisi B = {miring: .0f}cm")
print(f"Sisi C = {alas}cm")
print(f"Keliling = {keliling: .0f}cm")
print(f"Luas = {luas: .0f}cm")

File: PRAK110-2610817210017-Faris Ramadhan.c
#include <stdio.h>
#include <math.h>

int main (){
int tinggi = 12, alas = 5;

printf("Diketahui :\n");
printf("Alas = %dcm\n", alas);
printf("Tinggi = %dcm\n", tinggi);
printf("\n");
double miring = sqrt((alas * alas)+(tinggi * tinggi));
int keliling = (alas + tinggi + miring) ,luas = 0.5 * alas * tinggi;  
printf("Jawab :\n");
printf("Sisi A = %dcm\n", tinggi);
printf("Sisi B = %.0fcm\n", miring);
printf("Sisi C = %dcm\n", alas);
printf("Keliling = %dcm\n", keliling);
printf("Luas = %dcm\n", luas);
return 0;
}